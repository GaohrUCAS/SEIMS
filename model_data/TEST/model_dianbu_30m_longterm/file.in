MODE|Daily
INTERVAL|1,60
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-01-02 00:00:00