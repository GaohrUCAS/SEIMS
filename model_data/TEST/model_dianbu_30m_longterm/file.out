### INSTRUCTION, BY LIANG-JUN ZHU
### TYPE  : DATA AGGREGATION TYPE, e.g., SUM, AVE, MAX, NONE, etc. Types divided by ',', e.g., "SUM,AVE".
### STARTTIME : DATA DERIVED TIME START, FORMAT MUST BE "YYYY-MM-DD HH:MM:SS", e.g., 2011-01-01 00:00:00
### ENDTIME  : DATA DERIVED TIME END, SEE STARTTIME
### SUBBASIN : SELECTED SUBBASIN ID(s). 'ALL', 'OUTLET' ARE SELECTABLE, OR SUBBASIN ID NUMBERS SPLIT BY ','.
### FILENAME : OUTPUT FILENAME WITH SUFFIX, e.g., txt, asc, tif. FINAL FILENAME WILL BE APPENDED BY TYPE!
### USE   ï¼šOUTPUT OR NOT. DEFAULT IS 0, i.e., NOT USED. CHANGE TO 1 IF IT IS WANTED!

MODULE_CLASS	OUTPUTID	DESCRIPTION	UNIT	TYPE	STARTTIME	ENDTIME	INTERVAL	INTERVAL_UNIT	SUBBASIN	FILENAME	USE
TEST	CN2_M	Modify CN2	None	MEAN	2014-01-01 00:00:00	2014-01-02 00:00:00	-9999	-9999	ALL	CN2_M.tif	1
TEST	K_M	Modify K	mm/h	MEAN	2014-01-01 00:00:00	2014-01-02 00:00:00	-9999	-9999	ALL	K_M.tif	1